clientd3d/d3drender.c

```diff
@@ -455,7 +455,7 @@ void D3DRenderBegin(room_type *room, Draw3DParams *params)
 	Bool draw_particles = can_see;
 	Bool draw_background_overlays = can_see;
 
-	if (D3DRenderSkyBoxBegin(&current_room))
+	if (D3DRenderUpdateSkyBox(current_room.bkgnd))

```

Comment:
This uses the global extern "current_room", but there is also a parameter "room" to this function.  Can we get rid of the extern?